app/src/main/java/com/sopt/now/compose/data/NetworkModule.kt

```diff
@@ -0,0 +1,52 @@
+package com.sopt.now.compose.data
+
+import android.util.Log
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import com.sopt.now.compose.BuildConfig
+import com.sopt.now.compose.data.remote.service.AuthService
+import com.sopt.now.compose.data.remote.service.FriendService
+import com.sopt.now.compose.data.remote.service.UserService
+import kotlinx.serialization.json.Json
+import okhttp3.Interceptor
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+
+object NetworkModule {
+    private const val CONTENT_TYPE = "application/json"
+    private val json: Json =
+        Json {
+            ignoreUnknownKeys = true
+        }
+
+    private fun getLogOkHttpClient(): Interceptor {
+        val loggingInterceptor =
+            HttpLoggingInterceptor { message ->
+                Log.d("Retrofit2", "CONNECTION INFO -> $message")
+            }
+        loggingInterceptor.level = HttpLoggingInterceptor.Level.BODY
+        return loggingInterceptor
+    }
+
+    private val okHttpClient =
+        OkHttpClient.Builder()
+            .addInterceptor(getLogOkHttpClient())
+            .build()
+
+    fun provideRetrofit(url: String): Retrofit {
+        return Retrofit.Builder()
+            .baseUrl(url)
+            .client(okHttpClient)
+            .addConverterFactory(json.asConverterFactory(CONTENT_TYPE.toMediaType()))
+            .build()
+    }
+
+    inline fun <reified T> create(url: String): T = provideRetrofit(url).create<T>(T::class.java)
+}
```

Comment:
```suggestion
    inline fun <reified T> create(url: String): T = provideRetrofit(url).create()
}
```

타입 클래스를 넘겨주지 않아도 됩니다.

Reply:
```kotlin
inline fun <reified T> create(url: String): T = provideRetrofit(url).create(T::class.java)
```
는 적어줘야 오류가 안나는군요!!!